### PR TITLE
limits settings for debian

### PR DESCRIPTION
**Pull request by sp3c73r2038**

### CODE REVIEW

File: tasks/kernel.yml [hunk 1]
```diff
@@ -34,19 +34,22 @@
...
-#    insertafter='^export PATH'
-#    line='ulimit -n 65536'
-#    state=present
-#  when: ansible_os_family == 'Debian'
+# see how to set default limit system-wide for systemd units
+# https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html
+# https://serverfault.com/questions/958760/changing-ulimit-on-systemd
+- name: 'default open file limit for systemd units'
+  lineinfile:
+    dest=/etc/systemd/system.conf
```
Comment (pallxk) [inline tasks/kernel.yml]: Suggest to put this into `system.conf.d/*.conf`, as recommended by systemd.
https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#:~:text=Using%20drop%2Dins%20for%20local%20configuration%20is%20recommended%20over%20modifications%20to%20the%20main%20configuration%20file.

Comment (sp3c73r2038) [reply tasks/kernel.yml]: thanks for the advice.

use `/etc/systemd/system.conf.d/systemd_openfile.conf` instead.

File: tasks/kernel.yml [hunk 2]
```diff
@@ -26,29 +26,34 @@
...
-#    insertafter='^export PATH'
-#    line='ulimit -n 65536'
-#    state=present
-#  when: ansible_os_family == 'Debian'
+# see how to set default limit system-wide for systemd units
+# https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html
+# https://serverfault.com/questions/958760/changing-ulimit-on-systemd
+- name: 'default open file limit for systemd units'
+  template:
+    dest: /etc/systemd/system.conf.d/openfile.conf
```
Comment (pallxk) [inline tasks/kernel.yml]: Another suggestion is to prefix all filenames in those subdirectories with a two-digit number and a dash, to simplify the ordering of the files.

https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#:~:text=It%20is%20recommended%20to%20prefix%20all%20filenames%20in%20those%20subdirectories%20with%20a%20two%2Ddigit%20number%20and%20a%20dash%2C%20to%20simplify%20the%20ordering%20of%20the%20files.

But I think it's okay as it currently stands. Leave the decision to you.

**Comment by pallxk** [comment]
LGTM.

File: tasks/kernel.yml (hunk 2 above)
Comment (sp3c73r2038) [reply tasks/kernel.yml]: good catch, updated.

**Pull request merged**